common/src/test/java/gov/cms/ab2d/common/util/Coverage.java

```diff
@@ -8,23 +8,29 @@
 @Getter
 public class Coverage {
 
+    private final int coveragePeriod;
 
-    private final Long id;
+    private final long searchEventId;
 
-    private final int periodId;
+    private final String contract;
 
-    private final long searchEventId;
+    private final int year;
+
+    private final int month;
 
-    private final String beneficiaryId;
+    private final Long beneficiaryId;
```

Comment:
This should always be present, correct?  It is not possible to have a mapping to a beneID without a beneId.  

Better to make primitive.